python_modules/automation/automation/docker/images/dagster-base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE
+FROM "${BASE_IMAGE}"
+
+ARG DAGSTER_VERSION
+
+# ==> Dagster base layer
+RUN \
+# Cron - remove along with dagster-cron once daemon scheduler is default
```

Comment:
I may be wrong, but I think this is the case now: we can remove the `apt-get install cron` and `pip install dagster-cron` below.

Reply:
Removed